src/Juno/Runtime/Ledger.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE RecordWildCards #-}
+module Juno.Runtime.Ledger
```

Comment:
I try not to be a stickler about names but this (module, type, etc...) really can't be called Ledger as it is misleading -- there really is a Ledger but this is not it. The Ledger is the summation of the log as it is applied to the state machine and is held in the SM's MVar. Please rename.